packages/lib-react-components/src/ZooHeader/components/SignedInUserNavigation/SignedInUserNavigation.js

```diff
@@ -58,13 +58,13 @@ export default function SignedInUserNavigation (props) {
         tag='nav'
       >
         <NavListItem
-          color={unreadNotifications ? zooTheme.global.colors.lightTeal : '#B2B2B2'}
+          color={unreadNotifications ? 'accent-2' : '#B2B2B2'}
```

Comment:
Should this be a named colour?

Reply:
I think this is the only place where `#B2B2B2` is being used, so I didn't bother.